libs/platform/src/lib/components/approval-flow/approval-flow.component.ts

```diff
@@ -90,6 +91,7 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
     _dir: string;
 
     private subscriptions = new Subscription();
+    private _onDestroy$: Subject<void> = new Subject<void>();
```

Comment:
please remove it since it's not used anymore

Reply:
done